packages/webdriverio/tests/scripts/isElementClickable.test.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @jest-environment node
```

Comment:
same

Reply:
Should be here, we have it for script tests

Reply:
well, it had to be `@jest-environment jsdom` so that there is no need to create our own `document` and `window`, but in these tests it's not needed because we mock everything